addons/pythonscript_repl/python_repl.py

```diff
@@ -38,14 +38,23 @@ def _enter_tree(self):
             self.output_line("and enable Io Streams Capture.")
 
     def _exit_tree(self):
```

Comment:
So exit_tree is not guaranteed to be called ????
I would consider this as a bug on Godot side, is there an issue opened about this ?

Reply:
TBH that may have been some confusion on my part.